.circleci/config.yml

```diff
@@ -230,14 +230,24 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+      - architect/push-to-docker:
+          username: "${QUAY_USERNAME}"
+          password: "${QUAY_PASSWORD}"
+          registry: "quay.io"
+          image: "quay.io/giantswarm/aws-operator"
```

Comment:
I don't like that repetition. We should take registry name from the image.

Reply:
I don't know if this would be valid for other repositories like aliyun. Those are 2 differents values and should stay this way for less confusion IMO:
`registry` is the url used for login
`image` is the docker image name

Reply:
We checked and it's valid.